src/components/UI/Icons/ArrowIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from "react";
+import { IconProps } from "@/interfaces/IconInterface";
+
+export const ArrowIcon: FC<IconProps> = ({
+  width = "12",
+  height = "12",
+  color = "#ffffff",
+  disabled = false,
+}) => {
+  return (
+    <svg
+      width={width}
+      height={height}
+      viewBox={`0 0 ${width} ${height}`}
```

Comment:
```suggestion
      viewBox={viewBox}
```

I gueess viewBox shouold be passed as a whole prop, as its not always that it can be same format , what if it will be `0 0 0 ${width} ${height}` , or any other random form